languages/go/INSTRUCTIONS.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+This guide is for developers who want to use the Bitwarden Go SDK module in their own Go projects.
+
+The Bitwarden SDK is written in Rust, and we utilize FFI via [cgo](https://pkg.go.dev/cmd/cgo). We currently support static linking the Bitwarden Go module on Linux (x86-64 & arm64), Mac (x86-64 & arm64), and Windows (x86-64).
```

Comment:
```suggestion
The Bitwarden Go SDK module utilizes FFI calls to the Rust Bitwarden SDK via [cgo](https://pkg.go.dev/cmd/cgo).

The Bitwarden Go SDK module provides the following statically linked targets:

- Linux (x86-64 & arm64)
- macOS (x86-64 & arm64)
- Windows (x86-64)
```

I personally found the subject of this information unclear.
 
I think we are trying to give the reader more information on why cgo is needed?

Reply:
Yes, I think it's important to mention that we only support static linking and which targets we support, which brings in the question of what platforms we support. I definitely want it to be concise and clear though.

I've made some changes in https://github.com/bitwarden/sdk/pull/861/commits/730bbce055d05eeaa854d0cd405e5c1edde63953, hopefully that is more clear. Let me know what you think.